x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -424,3 +425,68 @@ SETTING_WS
     : WS -> channel(HIDDEN)
     ;
 
+
+//
+// METRICS command
+//
+mode METRICS_MODE;
+METRICS_PIPE : PIPE -> type(PIPE), popMode;
+
+fragment METRICS_UNQUOTED_IDENTIFIER_PART
+    : ~[=`|,[\]/ \t\r\n]
+    | '/' ~[*/] // allow single / but not followed by another / or * which would start a comment
+    ;
```

Comment:
No need to define this since it's already defined in the FROM command (line [196](https://github.com/elastic/elasticsearch/pull/108115/files#diff-e78771d02134981fe1563249a0fbbdabf95b40299b2086dd19fccef6e87f39c4R196))